node/tm-helpers/README.md

```diff
@@ -0,0 +1,13 @@
+# Eclipse Thingweb - TM Helpers
+
+The package provides helper tools for Thing Models.
```

Comment:
explaining what can be done would be nice

Reply:
Updated [here](https://github.com/eclipse-thingweb/td-tools/pull/27/commits/1466ee625d181dc9703d74aec18cde44f98ff3fc).